### PR TITLE
During investigating issue 7 on logstash-codec-json_lines, I refactored this a bit

### PR DESCRIPTION
**Pull request by guyboertje**
create eventify method
remove multiple public statements
change predicate order for better short-circuit eval

### CODE REVIEW

**Comment by jordansissel** [comment]
LGTM. Haven't tested, though.

**Comment by guyboertje** [comment]
@purbon - please review

**Comment by purbon** [comment]
please jenkins, test this.

File: lib/logstash/codecs/line.rb [hunk 1]
```diff
@@ -30,28 +31,30 @@ def register
...
     if !remainder.empty?
-      block.call(LogStash::Event.new("message" => @converter.convert(remainder)))
+      block.call(eventify(remainder))
     end
   end
 
-  public
   def encode(event)
-    if event.is_a? LogStash::Event and @format
+    if !@format.nil? && event.is_a?(LogStash::Event)
```
Comment (purbon) [inline lib/logstash/codecs/line.rb]: I like the idea of adding parentheses, but why changing ```@format```  for the new form? I understand both are good understood.

Comment (guyboertje) [inline lib/logstash/codecs/line.rb]: It is to do short-circuit evaluation as `@format` is crucial to the branch logic, if LHS is false, ruby will not do the RHS.

**Comment by guyboertje** [comment]
Redundant PR, closing.

**Pull request closed without merging**